tests/queries/0_stateless/03166_from_readable_size.sql

```diff
@@ -0,0 +1,27 @@
+-- Should be the inverse of formatReadableSize
+SELECT formatReadableSize(fromReadableSize('1 B'));
+SELECT formatReadableSize(fromReadableSize('1 KiB'));
+SELECT formatReadableSize(fromReadableSize('1 MiB'));
+SELECT formatReadableSize(fromReadableSize('1 GiB'));
+SELECT formatReadableSize(fromReadableSize('1 TiB'));
+SELECT formatReadableSize(fromReadableSize('1 PiB'));
+SELECT formatReadableSize(fromReadableSize('1 EiB'));
+
+-- Should be the inverse of formatReadableDecimalSize
+SELECT formatReadableDecimalSize(fromReadableSize('1 B'));
+SELECT formatReadableDecimalSize(fromReadableSize('1 KB'));
+SELECT formatReadableDecimalSize(fromReadableSize('1 MB'));
+SELECT formatReadableDecimalSize(fromReadableSize('1 GB'));
+SELECT formatReadableDecimalSize(fromReadableSize('1 TB'));
+SELECT formatReadableDecimalSize(fromReadableSize('1 PB'));
+SELECT formatReadableDecimalSize(fromReadableSize('1 EB'));
```

Comment:
Is it case insensitive? If not - let's do it.

Reply:
Nope, some of the lowercase letters have different meanings within the same context eg `m` -> `milli`, `b` -> `bit` so I opted to go for strict casing initially. Will change it then.

Thanks!

Reply:
Done!